uxarray/grid/utils.py

```diff
@@ -169,3 +169,75 @@ def _newton_raphson_solver_for_gca_constLat(init_cart,
         _iter += 1
 
     return np.append(y_new, constZ)
+
+
+def _get_cartesian_face_edge_nodes(face_nodes_ind, face_edges_ind,
+                                   edge_nodes_grid, node_x_values,
+                                   node_y_values, node_z_values):
+    """Helper function that constructs the face-edge connectivity for a Cartesian grid.
+    Parameters
+    ----------
+    face_nodes_ind :
+        The ith entry of Grid.face_node_connectivity
+    face_edges_ind :
+        The ith entry of Grid.face_edge_connectivity
+    edge_nodes_grid :
+        The entire Grid.edge_node_connectivity
+    node_x_values :
+        The values of Grid.node_x
+    node_y_values :
+        The values of Grid.node_y
+    node_z_values :
+        The values of Grid.node_z
+    Returns
+    -------
+    face_edges : np.ndarray
+        Face edge connectivity for Cartesian grid
+    """
```

Comment:
This is no longer considered connectivity, since it stores the coordinate values of the edge of each face as opposed to the index.

Name is fine, but description should reflect that it stores the coordinate values of each face edge.